.eslintrc.js

```diff
@@ -1,86 +1,24 @@
 module.exports = {
-	root: true,
-	env: {
-		browser: true,
-		es6: true,
-		node: true,
-		jest: true
-	},
 	globals: {
 		t: true,
 		n: true,
 		OC: true,
 		OCA: true,
 		OCP: true,
-		'$': true,
-		'_': true,
+		$: true,
+		_: true,
 		Vue: true,
-		VueRouter: true
+		VueRouter: true,
 	},
-	parserOptions: {
-		parser: 'babel-eslint',
-		ecmaVersion: 6
+	settings: {
+		'import/resolver': {
+			node: {
+				paths: ['src'],
+				extensions: ['.js', '.vue'],
+			},
+		},
```

Comment:
This part was already included in the config 
https://github.com/nextcloud/eslint-config/blob/f36fb05ab5d3a42618fb6542524f89b6883e29d1/index.js#L24

I added it because of the typescript, but maybe you don't lint it?

Reply:
Yeah those are not linted at the moment as I had some issues with regular eslint rules not working properly back then with the type identifiers, probably worth to have another look but let's keep that out of that PR for now. I've pushed another commit to get rid of the import resolver config for now then.